packages/matchbox/src/components/Pager/Pager.js

```diff
@@ -1,21 +1,33 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { margin } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+import styled from 'styled-components';
+
 import Next from './Next';
 import Previous from './Previous';
-import styles from './Pager.module.scss';
 
-class Pager extends Component {
-  static displayName = 'Pager';
+const StyledPager = styled('div')`
+  ${margin}
+`;
 
-  static Next = Next
-  static Previous = Previous
+function Pager(props) {
+  const { children, ...rest } = props;
 
-  render() {
-    return (
-      <div className={styles.Pager}>
-        {this.props.children}
-      </div>
-    );
-  }
+  return (
+    <StyledPager {...rest} display="inline-block">
```

Comment:
Since we moved this to a 'div`, `display` won't be available. you can move this up into the `styled` though